pkgs/development/libraries/glibc/common.nix

```diff
@@ -182,6 +182,14 @@ stdenv.mkDerivation ({
       "libc_cv_as_needed=no"
     ] ++ lib.optional withGd "--with-gd";
 
+  # Glibc's configuration machinery cannot parse *-muslabi64, and
+  # fails with a very cryptic error message.  This goes in
+  # extraConfigureFlags so it comes after the --host= added by stdenv,
+  # thereby overriding it.
+  extraConfigureFlags = lib.optionals (stdenv.hostPlatform.isMusl && stdenv.hostPlatform.isMips64n64 && stdenv.hostPlatform.isLittleEndian) [
+      "--host=mips64el-linux-musl"
+  ];
+
```

Comment:
Skip this for now please. Then this is 0 rebuilds and we can go straight to master.

(Also, a better approach is to rely on our gnu-config updating I think? We can worry about that in a subsequent PR.)

Reply:
Done; PR for this change is now #163401.

I have also changed the base to master as you suggested.  Let me know if I should change it back.